packages/ozone/src/api/moderation/emitEvent.ts

```diff
@@ -160,7 +162,10 @@ const handleModerationEvent = async ({
       ctx.cfg.service.did,
       moderationTxn,
     )
-    await tagService.evaluateForSubject()
+    const initialTags = isModEventReport(event)
+      ? [getTagForReport(event.reportType)]
+      : []
```

Comment:
Nit: No need to create empty unused array here.
```suggestion
      : undefined
```